test/test_distributions.py

```diff
@@ -366,6 +366,20 @@ def get_sp_dist(jax_dist):
         np.broadcast_to(np.identity(3), (2, 3, 3)),
         None,
     ),
+    T(
+        dist.CAR,
+        0.5,
+        2,
+        np.array(
+            [
+                [0.0, 1.0, 1.0, 0.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [1.0, 0.0, 0.0, 1.0],
+                [0.0, 1.0, 1.0, 0.0],
+            ]
+        ),
+        None,
```

Comment:
nit: no need for the last `None`.

It is better to add various test cases with different batch shapes for `loc, alpha, tau`. It seems one parameter value is missing here? (I guess we need 4 parameters).

Reply:
I'll write some tests once we've finalised the CAR distribution class

Reply:
I think we just need to add a few test cases here:
+ a vector of loc, a vector of alpha, a scalar of tau, a numpy matrix with is_sparse=True
+ a vector of loc, a scalar alpha, a vector of tau, a sparse matrix with is_sparse=False
+ a vector of loc, a scalar alpha, a scalar of tau,, a batch of numpy matrices with is_sparse=True
+ a batch of vectors of loc, a vector of alpha, a vector of tau, a batch of numpy matrices with is_sparse=False

Reply:
I'm not good at thinking about batching. Would you be able to show one of the examples with batching and I'll do the rest. Thanks

Reply:
I just added a couple of patterns in a comment below. Please mix them as desired :)